inc/type.class.php

```diff
@@ -1928,6 +1928,13 @@ static function install(Migration $migration) {
 
       $notepad = new Notepad();
       foreach ($allGenericObjectTypes as $genericObjectType => $genericObjectData) {
+         $itemtype = $genericObjectData['itemtype'];
+         if (! class_exists($itemtype, true)) {
+            $warning = "Unable to load the class '$itemtype'\n";
```

Comment:
This is usually not translated. I'm even not sure translations in the plugin's domain are available during upgrade.

I may try and if it works add a __()